src/Lykke.Service.BlockchainWallets.AzureRepositories/FirstGenerationBlockchainWalletRepository.cs

```diff
@@ -212,6 +214,19 @@ public async Task InsertOrReplaceAsync(IBcnCredentialsRecord credsRecord)
             await _bcnClientCredentialsWalletTable.InsertOrReplaceAsync(byAssetAddressEntity);
         }
 
+        public Task EnumerateBcnCredsByChunksAsync(string assetId, Func<IEnumerable<IBcnCredentialsRecord>, Task> chunks)
+        {
+            var partition = FirstGenerationBlockchainWalletEntity.FromBcnClientCredentials.ByAssetAddress.GeneratePartition();
+            
+            var filter = TableQuery.CombineFilters(
+                    TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.Equal, partition),
```

Comment:
You can use ```nameof(FirstGenerationBlockchainWalletEntity.PartitionKey)``` and ```nameof(FirstGenerationBlockchainWalletEntity.AssetId)``` instead of strings